test/ck_tile/gemm_multi_d/test_gemm_multi_d_util.hpp

```diff
@@ -16,41 +16,25 @@
 struct ElementWiseAddAdd
 {
     template <typename E, typename C, typename D0, typename D1>
-    CK_TILE_DEVICE auto operator()(E& e, const C& c, const D0& d0, const D1& d1) const -> void
+    __host__ __device__ auto operator()(E& e, const C& c, const D0& d0, const D1& d1) const -> void
     {
         const float x0_f = ck_tile::type_convert<float>(c) + ck_tile::type_convert<float>(d0) +
                            ck_tile::type_convert<float>(d1);
 
         e = ck_tile::type_convert<E>(x0_f);
     }
-
-    // TODO (mozga-amd): Required for the apply function in the reference; reuse or implement a
-    // function similar to the one used for device..
-    template <typename E, typename ParamT>
-    CK_TILE_HOST auto operator()(E& e, const ParamT& a) const -> void
-    {
-        e = e + ck_tile::type_convert<E>(a);
-    }
 };
 
 struct MultiplyMultiply
 {
     template <typename E, typename C, typename D0, typename D1>
-    CK_TILE_DEVICE auto operator()(E& e, const C& c, const D0& d0, const D1& d1) const -> void
+    __host__ __device__ auto operator()(E& e, const C& c, const D0& d0, const D1& d1) const -> void
```

Comment:
You have to use CK_TILE_HOST_DEVICE since host device can cause redefinition in ck tile